main/src/library/FileRead.flix

```diff
@@ -126,129 +126,129 @@ mod FileRead {
                     case ex: IOException          => Err(IoError(ErrorKind.Other, ex.getMessage()))
                 }
             }
 
-            def modificationTime(f, k) = {
+            def modificationTime(file, k) = {
                 try {
-                    k(fileAttributes(Paths.get(f)).lastModifiedTime().toMillis())
+                    k(fileAttributes(Paths.get(file)).lastModifiedTime().toMillis())
                 } catch {
                     case ex: InvalidPathException => Err(IoError(ErrorKind.InvalidPath, ex.getMessage()))
                     case ex: IOException          => Err(IoError(ErrorKind.Other, ex.getMessage()))
                 }
             }
 
-            def size(f, k) = {
+            def size(file, k) = {
                 try {
-                    k(Files.size(Paths.get(f)))
+                    k(Files.size(Paths.get(file)))
                 } catch {
                     case ex: InvalidPathException => Err(IoError(ErrorKind.InvalidPath, ex.getMessage()))
                     case ex: IOException          => Err(IoError(ErrorKind.Other, ex.getMessage()))
                 }
             }
 
-            def read(f, k) = {
+            def read(file, k) = {
                 try {
-                    k(new JString(Files.readAllBytes(Paths.get(f))))
+                    k(new JString(Files.readAllBytes(Paths.get(file))))
                 } catch {
                     case ex: InvalidPathException => Err(IoError(ErrorKind.InvalidPath, ex.getMessage()))
                     case ex: IOException          => Err(IoError(ErrorKind.Other, ex.getMessage()))
                 }
             }
 
-            def readLines(f, k) = {
+            def readLines(file, k) = {
                 try {
-                    k(ToFlix.toFlix(Files.readAllLines(Paths.get(f))))
+                    k(ToFlix.toFlix(Files.readAllLines(Paths.get(file))))
                 } catch {
                     case ex: InvalidPathException => Err(IoError(ErrorKind.InvalidPath, ex.getMessage()))
                     case ex: IOException          => Err(IoError(ErrorKind.Other, ex.getMessage()))
                 }
             }
 
-            def readBytes(f, k) = {
+            def readBytes(file, k) = {
                 try {
-                    k(Array.toVector(Files.readAllBytes(Paths.get(f))))
+                    k(Array.toVector(Files.readAllBytes(Paths.get(file))))
                 } catch {
                     case ex: InvalidPathException => Err(IoError(ErrorKind.InvalidPath, ex.getMessage()))
                     case ex: IOException          => Err(IoError(ErrorKind.Other, ex.getMessage()))
                 }
             }
 
-            def list(f, k) = {
+            def list(file, k) = {
                 try {
                     // TODO: Use java.nio.file.Files.list when we have support for Java generics.
-                    let file = new File(f);
+                    let file = new File(file);
```

Comment:
This indicates to me that something is wrong. `file` shadows `file` here.

We probably need to be sure to add the fparams to the environment before visiting the body of the handler.